### ISSUE TITLE
installing ShinyDash with Shiny Server

### ISSUE THREAD

**Issue by Guiui21** [trestletech/ShinyDash#39]
Hi,

I'm trying to download ShinyDash on my shiny server on CentOS. My problem is that with shiny server I have to install the package with root privilege so I have to do it through the terminal. When I do that, I get this error:

Error in contrib.url(repos, type) :
trying to use CRAN without setting a mirror

What should I do ?

Thanks.

**Comment by midraed** [comment]
Try specifying a repo before with options():

```
options("repos"="http://cran.us.r-project.org") 
# and them:
 install.packages("devtools")
```

**Comment by Guiui21** [comment]
Yes but my problem is on installing ShinyDash and not devtools. I can install devtools with no problem but when i do sudo R -e "devtools::install_github("ShinyDash", "trestletech")", it doesn't work

**Comment by Guiui21** [comment]
I have this problem for several packages from github

**Comment by midraed** [comment]
ShinyDash depends on shiny and XML, can you verify you have them installed?

Also, can you try running a R console in your server, and running the command `devtools::install_github("ShinyDash", "trestletech")` from the console?

**Comment by Guiui21** [comment]
So, I have shiny and XML installed. 
And when I run that code in my R console it works fine.
But still not working on my terminal.

**Comment by Guiui21** [comment]
I take it back, I didn't have XML install correctly. Thank you !

**Comment by midraed** [comment]
Excellent! @trestletech can close this now!

**Comment by James816322** [comment]
I have a question that when I run the command that
 **devtools::install_github("ShinyDash", "trestletech")**,
it cannot works and the error is
 **Error in parse_repo_spec(repo) :  Invalid git repo specification: 'ShinyDash'**,
I don't know why,Could you give me some advice?

**Comment by tasukuigarashi** [comment]
@James816322 The following code would work:

```
devtools::install_github("trestletech/ShinyDash")
```
